discovery/src/main/java/com/ibm/watson/discovery/v1/model/CredentialDetails.java

```diff
@@ -39,7 +39,7 @@
     /** saml. */
     String SAML = "saml";
     /** username_password. */
-    String USERNAME_PASSWORD = "username_password"; // pragma: whitelist secret
```

Comment:
I think you will run into issues locally if you don't set this. did you run the detect secret tool?

Reply:
fixed in [9b4fa35e968ec895fe02e1a6d5adfd529f3cdd5f](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/9b4fa35e968ec895fe02e1a6d5adfd529f3cdd5f)